src/pages/_document.js

```diff
@@ -3,10 +3,28 @@ import Document, { Head, Main, NextScript } from 'next/document'
 
 export default class DefaultDocument extends Document {
   render() {
+    const FB_SCRIPT = `
+      window.fbAsyncInit = function() {
+        FB.init({
+          appId      : '240115020081725',
+          xfbml      : true,
+          version    : 'v3.0'
+        });
+        FB.AppEvents.logPageView();
+      };
+      (function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = 'https://connect.facebook.net/pt_BR/sdk.js#xfbml=1&version=v3.0&appId=240115020081725&autoLogAppEvents=1';
```

Comment:
Same here, this appID should be a env var so it's changed in the app configurations.